pdr_backend/lake/etl.py

```diff
@@ -123,14 +179,16 @@ def do_bronze_step(self):
         # let's keep track of time passed so we can log how long it takes for this step to complete
         st_ts = time.time_ns() / 1e9
 
-        self.update_bronze_pdr()
+        self.do_bronze_queries()
```

Comment:
are these queries or are they in fact statements? Can you please check the usage everywhere? Queries implies that you ask the database for information, statements imply you are asking the database **to do something**

Reply:
these queries are doing a bunch of work... from fetching, to joining, to many other actions. they then use the results to build additional rows, tables, etc...

i'd like to leave this named as-is